docroot/modules/custom/uiowa_core/uiowa_core.module

```diff
@@ -134,6 +134,20 @@ function template_preprocess_uiowa_core_headline(&$variables) {
 
 }
 
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function uiowa_core_preprocess_media(&$variables) {
+  $view_mode = $variables['view_mode'];
+  $accepted_view_modes = ['large', 'medium', 'small'];
+  foreach ($accepted_view_modes as &$value) {
```

Comment:
```suggestion
  foreach ($accepted_view_modes as $value) {
```
Since `$value` is not being modified in the loop, there is no need to pass it by reference.

Reply:
Ah, thank you